Packs/SafeNet_Trusted_Access/IncidentFields/incidentfield-SafeNet_Trusted_Access_-_High_Risk_Group.json

```diff
@@ -0,0 +1,27 @@
+{
+    "associatedToAll": true,
```

Comment:
If it associate to one incident, lets write it like this (example): 
`    "associatedToAll": false,
    "associatedTypes": [
        "Infinipoint Compliant",
        "Infinipoint NotCompliant"
    ],`

Reply:
Done for all the incident fields.